distro/configuration/drugs/drugs.csv

```diff
@@ -14,11 +14,16 @@ e6bd712f-5fb7-4704-8cc3-f26b7f710d20,,Lisinopril 10 mg,78945AAAAAAAAAAAAAAAAAAAA
 1e5421d1-abcd-4d88-8776-1a8057506aba,,Lisinopril 20 mg,78945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,20mg,
 f6ad6ed0-953c-4b45-ba41-2f302f3b8b09,,Lisinopril 30 mg,78945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,30mg,
 41323ed9-967b-4afa-bc9b-cda3ceb42050,,Lisinopril 40 mg,78945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,40mg,
+43615798-db46-4e2a-95a8-182ead4da45b,,Amlodipine,71137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,
+00a789ff-da44-42b9-b929-3a069d98d3e1,,Amlodipine 5mg,71137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5mg,
 762b4267-6ce6-4de7-8342-7d415fedda6b,,Levothyroxine sodium,78809AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,
 15c7d21f-cd1a-4eed-ab79-2fcfe0d99592,,Levothyroxine sodium 75 mcg,78809AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,75mcg,
+a80acf74-a85c-4fef-8dc4-d763e364663f,,Metformin,79651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,
 81dadd82-3ac8-4aab-9f61-848b89b3f053,,Metformin Co,79651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,
 721497f0-dcf4-4f05-8fe7-e2e487b0d774,,Metformin Co 500mg,79651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, 500mg,
 d590f509-1e79-479c-bcdd-5f3812b16803,,Metformin Co 1000mg,79651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, 1000mg,
+405bf055-32ff-405c-b041-4e4543f66e99,,Metformin 1000mg,79651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1000mg,
+095f2aaa-351b-4848-9039-87bf67e2e97b,,Metformin Co 500mg,63aaaeba-9b3c-4ff5-9faf-522c10b6f54c,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,500mg,
```

Comment:
Shouldn't this be pointing to the same concept drug as the other Metformin Co?

Reply:
Yeah - that one is a legacy introduced by one of the Mekom team members; I don't know why it has a different UUID. I was afraid to mess something up for them - all I did here was move line 26 up from below (so it lives w/ the other Metformins). Do you know - if I change the UUID for that one to be the correct CIEL code, would that screw up something in Ozone?

Reply:
@rbuisson Question for you ^^

Reply:
But @ibacher it doesn't make any sense to have twice "Metformin Co 500mg" right? (It was already [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/696/files#diff-b253c8b7450e01411f38bfdcc20dc2dee28bd1c77fac115618cdc32896ee236bR23)).

What matters for us is the _drug_ UUID because it is used as joint univocal reference between the EMR system and the pharmacy system.

Reply:
Whoops... I missed that we had that duplicated as well. Ok... So as long as the drug itself has the same UUID, it should be fine?

Reply:
Yes, however we may also override drugs.csv within Ozone to hedge ourselves against such upstream changes. This is not yet the case right now though.

Reply:
I understand the problem now - the problem is that we've / Ozone / etc have been managing Drug Concepts here in [this drugs_concepts.csv](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/main/distro/configuration/concepts/drugs_concepts.csv) _instead of_ in OCL, which is why there is this random UUID for Metformin instead of a CIEL code. I will change this promptly for O3 and have started here: https://github.com/openmrs/openmrs-distro-referenceapplication/pull/700
Eventually I will use OCL to deprecate and remove the drugs_concepts.csv altogether.

Reply:
W.r.t. not impacting Ozone - @mks-d what we need to get to next is, (1) slightly separate this "Basic EMR" metadata config from the refapp directly, and (2) versioning this "Basic EMR" metadata package. This would give you control for whether or not you want Ozone to use this Basic EMR metadata config at all, and if you do, you'd have version control just like for any esm-app.